README.rst

```diff
@@ -41,6 +43,13 @@ Viewer0D
 
 * **Keithley_Pico**: Pico-Amperemeter Keithley 648X Series, 6430 and 6514
 * **Keithley2110**: Multimeter Keithley  2110
+* **Keithley27XX**: Keithley 27XX Multimeter/Switch System using SCPI commands (tested with models 2700 & 2701)
+
+
```

Comment:
the description below should be added into another paragraph such as the 
```
Infos
=====
```

one

Reply:
Thanks for the review, this description was not up to date since made for my old configuration process. Now deleted.